### DISCUSSION TITLE
Quick Fixes

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uAssets#12678] [Q&A] [answered]
Hi. I'm just wondering - how long should filters stay in that list before they get moved to normal ones?

A huge portion of the list is `wp.pl` (and other related websites). I've looked through the commits and there haven't been major changes to them in a while... And since these belong to a local list, not a global one, it seems a bit strange keeping them in Quick for that long.

Also, some of these filters seem rather inefficient. Mainly starting with this commit https://github.com/uBlockOrigin/uAssets/commit/6f92be9cb5e8f9ac44f6cca1b22f9456e4b36efc which introduced `:has-text(Zaloguj)` to (nearly?) every single one of them.
I understand there might be some frontend differences between the websites making this way easier for "grouping" filters, but... should the lower number of lines take priority over the efficiency of a single filter?

Also, why even limit the filters to people who aren't logged in in the first place? I made a test account on wp and obviously none of the filters work when logged in. So I'm just really confused as to what was the reasoning behind this move.

Oh, and @JobcenterTycoon since you seem to be the main contributor, do you mind properly naming your commits? Having tens (which will soon turn to hundreds?) of commits named "update quick" is just really annoying to look through.
Why not continue with yesterday's naming ("update rjno1") and use that for all commits (obviously mentioning the correct website)? "Update last commit" would also be fine then, but "adjust filter" is just as clear as "update quick" is.
This should be keeping in line with [contributing>commit message](https://github.com/uBlockOrigin/uAssets/blob/master/CONTRIBUTING.md#commit-message). Since there is no issue linked with the details, the minimum info (IMO) should be mentioning the website.

**Comment by JobcenterTycoon** [comment, marked as answer]
Thanks for the feedback, i can remove `:has-text(Zaloguj)` and see what happens. My hope is the admin is logged in and not notice the filter (worked for tpu 14 days). 

Now i know if i just name it `update quick` don't change anythink, websites like `rjno1` still change fast so yes i will push less updates (not so many for one site like every hour) and clarify.

**Comment by ghost** [comment]
Some filters are months old https://github.com/uBlockOrigin/uAssets/blame/master/filters/quick-fixes.txt Need some cleanup.

**Reply by JobcenterTycoon** [reply]
Yes some filters are stable

- ~~wp.pl~~ (update: no)
- next-episode.net
- a few aternos.org filter
- techpowerup.com cosmetic filter

But first it need to be added to regular filter and after 4 days the quick filter can be removed
